### ISSUE TITLE
[BUG] After the first unsuccessful attempt to complete the captcha challenge, the remaining attempts become failures

### ISSUE THREAD

**Issue by flexstat** [bunkerity/bunkerweb#870]
### What happened?

Uses standard captcha challenge (USE_ANTIBOT=captcha) docker-compose and bunker 1.5.5

If the first attempt to complete the challenge is unsuccessful, all other attempts will fail.

If you pass the challenge the first time, then everything is ok

### How to reproduce?

To reproduce the situation, you need to select in the config - USE_ANTIBOT=captcha
Launch the bunker, wait for loading, then fail the first attempt of the challenge and try to complete the challenge again, after which all attempts will be unsuccessful, even if the captcha is entered correctly

### Configuration file(s) (yaml or .env)

```YAML
version: "3"
 
services:
  mybunker:
    image: bunkerity/bunkerweb:1.5.5
    ports:
      - 80:8080
      - 443:8443
    environment:
      - USE_ANTIBOT=captcha
      - API_WHITELIST_IP=127.0.0.0/8 15.20.30.0/24
      - SERVE_FILES=no
      - USE_CLIENT_CACHE=yes
      - USE_REVERSE_PROXY=yes
      - REVERSE_PROXY_URL=/
      - REVERSE_PROXY_HOST=http://app
      - |
        CUSTOM_CONF_HTTP_upstream.conf=
        upstream app {
          server ${SERVER_1}:80;
        }
      - USE_GZIP=yes
      - USE_BLACKLIST=no
      - USE_DNSBL=no
      - USE_BAD_BEHAVIOR=no
      - USE_MODSECURITY=no
      - USE_BUNKERNET=no
 
 
    labels:
      - "bunkerweb.INSTANCE=yes" # required for the scheduler to recognize the container
    networks:
      - bw-universe
      - bw-services
 
  bw-scheduler:
    image: bunkerity/bunkerweb-scheduler:1.5.5
    depends_on:
      - mybunker
    environment:
      - DOCKER_HOST=tcp://bw-docker-proxy:2375
    volumes:
      - ./bw-data:/data
    networks:
      - bw-universe
      - bw-docker
 
  bw-docker-proxy:
    image: tecnativa/docker-socket-proxy:nightly
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
    environment:
      - CONTAINERS=1
      - LOG_LEVEL=warning
    networks:
      - bw-docker
 
  app1:
    image: tutum/hello-world
    networks:
      - bw-services
 
  app2:
    image: tutum/hello-world
    networks:
      - bw-services
 
volumes:
  bw-data:
 
networks:
  bw-services:
  bw-universe:
    ipam:
      driver: default
      config:
        - subnet: 15.20.30.0/24
  bw-docker:
```

### Relevant log output

```shell
mybunker_1 | 1172.0.0.1 185.0.0.1 - - [15/Jan/2024:19:15:33 +0000] "GET /challenge HTTP/1.1" 200 81045 "http://172.0.0.1/challenge" "Mozilla/5.0 (X11 ; Ubuntu; Linux x86_64; rv:121.0) Gecko/20100101 Firefox/121.0"
mybunker_1 | 2024/01/15 19:15:41 [warn] 88#88: 935 [ANTIBOT] client failed challenge : wrong value, client: 185.0.0.1, server: www.example.com, request: "POST /challenge HTTP /1.1", host: "172.0.0.1", referrer: "http://172.0.0.1/challenge"
mybunker_1 | 2024/01/15 19:15:41 [notice] 88#88:935 [ACCESS] antibot returned status 0 : displaying challenge to client, client: 185.0.0.1, server: www.example.com, request: "POST /challenge HTTP/1.1", host: "172.0.0.1", referrer: "http://1172.0.0.1/challenge"
```

### BunkerWeb version

1.5.5

### What integration are you using?

Docker

### Removed private data

- [X] I have removed all private data from the configuration file and the logs

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fl0ppy-d1sk** [comment]
Hello @flexstat,

Fix should be available on the testing version, more info here : https://docs.bunkerweb.io/testing/

Don't hesitate to test it and tell us if everything is ok !

**Comment by fl0ppy-d1sk** [comment]
Hello @flexstat,

Fix is now available in the v1.5.6 release. Thanks for your feedback !
